src/view/drag-drop-context/drag-drop-context.jsx

```diff
@@ -150,31 +150,46 @@ export default class DragDropContext extends React.Component<Props> {
     let previous: State = this.store.getState();
 
     this.unsubscribe = this.store.subscribe(() => {
-      const previousValue: State = previous;
       const current = this.store.getState();
```

Comment:
The only change here was to move the style marshal update before the hooks are called. However, I wanted to add some explanation to what is going on because the timing of everything is really important